handlers/easy_info/__init__.py

```diff
@@ -1,10 +1,42 @@
 from eventhandler import EventHandler
 
+import re
+
+ASSIGN_MSG = 'assign me'
+
+MSG = ('Hi! If you have any questions regarding this issue, feel free to make'
+       ' a comment here, or ask it in the `#servo` channel in '
+       '[IRC](https://wiki.mozilla.org/IRC).\n\n'
+       'If you intend to work on this issue, then add `@%s: %s`'
+       ' to your comment, and I\'ll assign this to you. :smile:')
+
+RESPONSE_FAIL = ('It looks like this has already been assigned to someone.'
+                 ' I\'ll leave the decision to a core contributor.')
+
+RESPONSE_OK = ('Hey @%s! Thanks for your interest in working on this issue.'
+               ' It\'s now assigned to you!')
+
 
 class EasyInfoHandler(EventHandler):
     def on_issue_labeled(self, api, payload):
         if payload['label']['name'].lower() == 'e-easy':
-            api.post_comment(('Please make a comment here if you intend to '
-                              'work on this issue. Thank you!'))
+            api.post_comment(MSG % (api.user, ASSIGN_MSG))
+
+    def on_new_comment(self, api, payload):
+        if payload['issue']['state'] != 'open':
+            return
+
+        user = payload['comment']['user']['login']
```

Comment:
Let's make sure that the author is not the bot, because the MSG comment body contains the ASSIGN_MSG string and our automated testing doesn't cover this case.